tests/decorators/test_python.py

```diff
@@ -870,6 +870,19 @@ def org_test_func():
     ), "decorated function does not have __wrapped__ attribute"
     assert decorated_test_func.__wrapped__ is org_test_func, "__wrapped__ attr is not the original function"
 
+def test_task_decorator_has_doc_attr():
+    """
+    Test  @task original underlying function docstring
+    through the __doc__ attribute.
+    """
+
+    def org_test_func():
+        """Docstring"""
+
+    decorated_test_func = task_decorator(org_test_func)
+    assert hasattr(decorated_test_func, "__doc__"), "decorated function does not have __doc__ attribute"
+    assert decorated_test_func.__doc__ == org_test_func.__doc__, "__doc__ attr is not the original docstring"
```

Comment:
Nit: lets make the assert message makes sense when reading the code (without having to remember that it is only shown when it fails)


```suggestion
    assert hasattr(decorated_test_func, "__doc__"), "decorated function should have __doc__ attribute"
    assert decorated_test_func.__doc__ == org_test_func.__doc__, "__doc__ attr should be the original docstring"
```